Gemfile

```diff
@@ -6,7 +6,7 @@ ruby "2.6.5"
 gem "active_model_serializers"
 gem "api-pagination"
 gem "asciidoctor"
-gem "bibliothecary"
+gem "bibliothecary", "~> 6.12.2"
```

Comment:
Any reason to specify rather than just a bundle update.

Reply:
Strangely that wasn't updating the gem for me! I wonder if it's because it's a new minor version?

Reply:
May have been that rubygems didn't have it yet?

Reply:
Wait a sec, I tried again and it worked. I bet I was just running `bundle` before 🤦🏻